packages/slate-editor/src/modules/nodes-hierarchy/withNodesHierarchy.ts

```diff
@@ -18,7 +18,7 @@ export function withNodesHierarchy(schema: NodesHierarchySchema) {
             }
 
             for (const normalizer of normalizers) {
-                const isNormalized = normalizer(editor, path);
+                const isNormalized = normalizer(editor, node, path);
```

Comment:
We have `node` here, so we can pass it to the normalizers. No need to call `Node.get()` to re-retrieve the node in every normalizer.